python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -317,7 +324,7 @@ def fitBackground(self, maskedImage, nx=0, ny=0, algorithm=None):
 
         self.log.debug("Ignoring mask planes: %s", ", ".join(self.config.ignoredPixelMask))
         if (maskedImage.mask.getArray() & badMask).all():
-            raise pipeBase.NoWorkFound("All pixels masked. Cannot estimate background")
+            raise TooManyMaskedPixelsError("All pixels masked. Cannot estimate background")
```

Comment:
```suggestion
            raise TooManyMaskedPixelsError("All pixels masked. Cannot estimate background.")
```

Reply:
Can you squash away my commit. It was just for testing